plugins/train/trainer/_base.py

```diff
@@ -267,12 +268,12 @@ def load_generator(self):
                                           self.config)
         return generator
 
-    def train_one_batch(self, do_preview):
+    def train_one_batch(self):
         """ Train a batch """
         logger.trace("Training one step: (side: %s)", self.side)
-        batch = self.get_next(do_preview)
+        model_inputs, model_targets = self.get_next()
         try:
-            loss = self.model.predictors[self.side].train_on_batch(*batch)
+            loss = self.model.predictors[self.side].train_on_batch(x=model_inputs, y=model_targets)
```

Comment:
*batch makes code readability harder, breaking out inputs explicitly for clarity